src/main/webapp/view/discussion/discussion.js

```diff
@@ -200,20 +115,130 @@ function getNearbyDiscussionComments(timestampMs) {
   return nearby;
 }
 
+
+class DiscussionManager {
+  static #ENDPOINT = '/discussion';
+  static #PARAM_LECTURE = 'lecture';
+  static #PARAM_PARENT = 'parent';
+  static #PARAM_TIMESTAMP = 'timestamp';
+  static #PARAM_TYPE = 'type';
+  #lecture;
```

Comment:
Just curious, what does the `#` do?

Reply:
Marks a field as private, so supposedly it can only be accessed within the class.